File: Source/OregonTilth.Models/DataTransferObjects/Page/PageUpdateDto.cs
namespace OregonTilth.Models.DataTransferObjects.Page
{
    public class PageUpdateDto
    {
        public string PageName { get; set; }
        public int PageID { get; set; }
        public string PageContent { get; set; }
        public int? ParentPageID { get; set; }
    }
}

File: OregonTilth.Web/src/app/shared/models/page/page-update-dto.ts
import { PageDto } from "../generated/page-dto";

export class PageUpdateDto {
    PageID: number;
    PageName: string;
    ParentPageID: number;
    PageContent: string;
    SortOrder: number;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}
